### PR TITLE
updated  mkdocs-en.yml and en-homepage

### PR DESCRIPTION
**Pull request by windsonsea**
- 更新了英文目录
- 添加了英文首页

@SAMZONG  need your review and confirm how to implement the en homepage.

### CODE REVIEW

File: mkdocs-en.yml [hunk 1]
```diff
@@ -1,110 +1,557 @@
...
+            - Pipeline Concepts: zh/amamba/03UserGuide/Pipeline/PipelineConfig/PipelineConcept.md
+            - Job Template Parameter with GUI: zh/amamba/03UserGuide/Pipeline/PipelineConfig/PipelineStep.md
+      - What is Multi-Cloud Orchestration:
+        - Introduction:
+          - What is Multi-Cloud Orchestration: zh/kairship/01product/whatiskair.md
+          - Features: zh/kairship/01product/features.md
+          - Benefits: zh/kairship/01product/benefits.md
+          - Scenarios: zh/kairship/01product/scenarios.md
+          - Concepts: zh/kairship/01product/concepts.md
+        - Installation:
```
Comment (Michelle951) [inline mkdocs-en.yml:311]: may be we should consistently use either "install" as in line 99 or "installation" as in this line.

File: mkdocs-en.yml [hunk 2]
```diff
@@ -1,110 +1,557 @@
...
+          - Create with YAML: zh/kairship/05workload/yaml.md
+        - Resources:
+          - Multi-Cloud Service and Ingress: zh/kairship/06resource/ingress.md
+          - Multi-Cloud Namespace: zh/kairship/06resource/namespace.md
+          - Multi-Cloud Storage Claims: zh/kairship/06resource/storage.md
+          - Multi-Cloud ConfigMap and Secret: zh/kairship/06resource/configmap.md
+        - Policies:
+          - Deployment Policies: zh/kairship/07policy/deploy.md
+          - Differentiation Policies: zh/kairship/07policy/different.md
+      - Microservice Governance:
```
Comment (Michelle951) [inline mkdocs-en.yml:333]:
```suggestion
      - Microservice Engine:
```

File: mkdocs-en.yml [hunk 3]
```diff
@@ -1,110 +1,557 @@
...
+              - Import/Export Configuration: zh/skoala/registry/host/config-center/import-config.md
+              - Update Configuration: zh/skoala/registry/host/config-center/update-config.md
+              - Delete Configuration: zh/skoala/registry/host/config-center/delete-config.md
+              - Restore Configuration: zh/skoala/registry/host/config-center/restore-config.md
+              - Query Listener: zh/skoala/registry/host/config-center/search-listening.md
+              - View Sample Code: zh/skoala/registry/host/config-center/code-example.md
+            - Alerts:
+              - Service Alert: zh/skoala/registry/host/monitor&alert/monitor-service.md
+              - Resource Alert: zh/skoala/registry/host/monitor&alert/monitor-resource.md
+            - View Logs: zh/skoala/registry/host/log.md
```
Comment (Michelle951) [inline mkdocs-en.yml:368]:
```suggestion
            - Logs: zh/skoala/registry/host/log.md
```

File: mkdocs-en.yml [hunk 4]
```diff
@@ -1,110 +1,557 @@
+# Project information
 site_name: DaoCloud Enterprise
-# site_dir: 'public/en/'
+site_url: https://docs-alpha.daocloud.io/
+site_author: DaoCloud
+site_description: >-
+  DaoCloud Enterprise 5.0 is the next-generation PaaS platform, created with many leading cloud-native technologies, to easily promote your digital business.
```
[the remark this answers is not in the document]
Comment (windsonsea) [reply mkdocs-en.yml:6]: Not sure whether a newline is correct or not here.

**Review by Michelle951 — APPROVED** [review]

**Pull request merged**